.travis.yml

```diff
@@ -1,39 +1,106 @@
+sudo: false
+dist: trusty
 language: php
-
 notifications:
   email:
     on_success: never
     on_failure: change
-
-php:
-  - 5.6
-  - 7
-
-after_success: curl -L https://raw.githubusercontent.com/miya0001/travis2wpplugin/master/deploy.sh | bash
-
+cache:
+  directories:
+    - vendor
+    - $HOME/.composer/cache
+after_success: curl -L https://raw.githubusercontent.com/miya0001/travis2wpplugin/master/deploy.sh
+  | bash
 env:
   matrix:
-    - WP_VERSION=latest WP_MULTISITE=0
-    - WP_VERSION=nightly WP_MULTISITE=0
-
+  - WP_VERSION=latest WP_MULTISITE=0
```

Comment:
I think there is a missing tab, or does this indention work in YML?

Reply:
It works. But I can fix it regardless. Do the testing changes look good otherwise? It now tests under all supported versions of PHP, the PHPCS settings include PHP compatibility testing to avoid a version issue as previously, and it checks in two different ways for incorrect text domains(which I fixed several of as a result). I'm adding these improvements to all of my plugins.

Reply:
sure 👍